src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -605,52 +616,55 @@ def get_available_view_names(self, rom_name: str):
         self._log_key = "GetImageViewNames"
 
         if not self.evaluation_is_initialized:
-            msg = "TwinModel has not been initialized! "
-            msg += "Please initialize evaluation before to call this method!"
+            msg = "Twin model has not been initialized. "
+            msg += "Initialize the evaluation before calling this method."
             self._raise_error(msg)
 
         if self.tbrom_info is None:
-            self._raise_error("Twin model does not include any TBROM!")
+            self._raise_error("Twin model does not include any TBROM.")
 
         if rom_name not in self.tbrom_names:
-            msg = f"The provided rom_name {rom_name} has not been found in the available TBROM names. "
-            msg += f"Please call this method with a valid TBROM name."
-            msg += f"\n Available TBROM name are: {self.tbrom_names}"
+            msg = f"The provided ROM name {rom_name} has not been found in the available TBROM names. "
+            msg += f"Call this method with a valid TBROM name."
+            msg += f"\n Available TBROM names are: {self.tbrom_names}"
```

Comment:
Should there be a space before "Available"?

Reply:
nope, thanks!

Reply:
done